src/controllers/users-controller.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+const {
+  createUser,
+  getAllUsers,
+  getUserById,
+  deleteUserById,
+  UpdateUserData,
+} = require('../services/userService.js');
+
+const getUsers = async (req, res) => {
+  res.send(await getAllUsers());
+};
+
+const postUser = async (req, res) => {
+  const { name } = req.body;
+
+  if (typeof name !== 'string') {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = await createUser(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const getUser = async (req, res) => {
+  const userId = +req.params.userId;
+
+  const searchedUser = await getUserById(userId);
```

Comment:
What about 500 internal error. You should use try catch blocks on async actions.